pandas/core/indexes/accessors.py

```diff
@@ -126,6 +126,43 @@ class DatetimeProperties(Properties):
     """
 
     def to_pydatetime(self):
+        """
+        Return an ndarray of native Python datetime objects.
+
+        Timezone information is retained if present.
+
+        Returns
+        -------
+        numpy.ndarray
+            object dtype array containing native Python datetime objects.
+
+        Examples
+        --------
+        This method is available on both Series with datetime values, under the
+        ``.dt`` accessor, and directly on DatetimeIndex.
+
+        **Series**
+
+        >>> s = pd.Series(pd.date_range('20180310', periods=2))
+        >>> s.head()
+        0   2018-03-10
+        1   2018-03-11
+        dtype: datetime64[ns]
```

Comment:
Unnecessary `head()`.

Reply:
Hi @datapythonista ,
If I remove the head, the validation tests fails. 
Line 18, in pandas.Series.dt.to_pydatetime
Failed example:
    s = pd.Series(pd.date_range('20180310', periods=2))
Expected:
    0   2018-03-10
    1   2018-03-11
    dtype: datetime64[ns]
Got nothing

Reply:
fixed it by just using >>> s

Reply:
You can just do `s` by itself.